<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cracker</class>
 <widget class="QWidget" name="Cracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="hedding">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>201</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hash Cracker</string>
   </property>
  </widget>
  <widget class="QLabel" name="hash_labe">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hash</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hash">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>175</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>561</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>wordlist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8 digit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7 digit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 digit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 digit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 digit</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="crack_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>260</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Crack</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
